<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationTraceDialog</class>
 <widget class="QDialog" name="CalibrationTraceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calibration Measurements</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGroupBox" name="port1Standards">
       <property name="title">
        <string>Port 1 Standards</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="port1Male">
          <property name="text">
           <string>Male</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">port1Group</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="port1Female">
          <property name="text">
           <string>Female</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">port1Group</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="port2Standards">
       <property name="title">
        <string>Port 2 Standards</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="port2Male">
          <property name="text">
           <string>Male</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">port2Group</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="port2Female">
          <property name="text">
           <string>Female</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">port2Group</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="throughStandard">
       <property name="title">
        <string>Through Standard</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="throughCalkit">
          <property name="text">
           <string>From calibration kit</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">throughGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="throughZero">
          <property name="text">
           <string>Zero-length through</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">throughGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bMeasure">
       <property name="text">
        <string>Measure</string>
       </property>
       <property name="icon">
        <iconset theme="media-playback-start" resource="../icons.qrc">
         <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bDelete">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset theme="edit-delete" resource="../icons.qrc">
         <normaloff>:/icons/trash.png</normaloff>:/icons/trash.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bApply">
       <property name="text">
        <string>Apply Calibration</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/ok.png</normaloff>:/icons/ok.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="port1Group"/>
  <buttongroup name="port2Group"/>
  <buttongroup name="throughGroup"/>
 </buttongroups>
</ui>
